subprojects/docs/src/docs/release/notes.md

```diff
@@ -117,24 +103,10 @@ testing {
 }
 ```
 
-#### Dependency verification metadata improvements
+For more information about warning modes, see [JVM test suite](userguide/jvm_test_suite_plugin.html).
 
-The following nodes with dependency verification metadata file `verification-metadata.xml` now support a `reason` attribute:
 
-- the `trust` xml node under `trusted-artifacts`
-- the `md5`, `sha1`, `sha256` and `sha512` nodes under `component`
-
-#### Dependency verification CLI improvements
```

Comment:
@asodja can you add more context to this section, why is it helpful for a developer?

Reply:
Acknowledged

Reply:
I added reviews suggestions down below. PTAL.